spec/unit/services_inspector_spec.rb

```diff
@@ -71,11 +78,14 @@
       expect(summary).to eq("Found 3 services.")
     end
 
-    it "returns data about SysVinit services when no systemd is present" do
+    it "returns data about SysVinit services on a suse system when no systemd is present" do
       system = double
       expect(system).to receive(:run_command).
         with("systemctl", "--version").
         and_raise(Cheetah::ExecutionFailed.new(nil, nil, nil, nil))
+      expect(system).to receive(:run_command).
+        with("chkconfig", "--version").
+        and_raise(Cheetah::ExecutionFailed.new(nil, nil, nil, nil))
```

Comment:
That's because we need to find out if SUSE's or Red Hat's chkconfig is on the system. If we determined the OS by running the OS inspector first we could skip this step.